src/DynamoCore/DynamoCore.csproj

```diff
@@ -91,9 +91,8 @@ limitations under the License.
       <HintPath>..\packages\Newtonsoft.Json.8.0.3\lib\net45\Newtonsoft.Json.dll</HintPath>
       <Private>True</Private>
     </Reference>
-    <Reference Include="ProtoGeometry, Version=1.0.0.0, Culture=neutral, processorArchitecture=MSIL">
-      <SpecificVersion>False</SpecificVersion>
-      <HintPath>..\..\extern\ProtoGeometry\ProtoGeometry.dll</HintPath>
+    <Reference Include="ProtoGeometry, Version=2.7.0.9606, Culture=neutral, processorArchitecture=AMD64">
+      <HintPath>..\packages\DynamoVisualProgramming.LibG_226_0_0.2.7.0.9606\lib\net48\ProtoGeometry.dll</HintPath>
```

Comment:
@alfredo-pozo can we integrate the latest LibG nuget packages that would have version bumped up to 2.9 for the LibG binaries.

Reply:
Done with last commit